qt/aqt/editor.py

```diff
@@ -124,6 +136,7 @@ def __init__(
         self.last_field_index: int | None = None
         # current card, for card layout
         self.card: Card | None = None
+        self.state: EditorState | None = None
```

Comment:
Do we need both None and INITIAL states here?

Reply:
No, you're right, don't see much use in starting with `None`. Changed!